src/main/webapp/js/common/helper.es6

```diff
@@ -0,0 +1,73 @@
+/**
+ * Tests whether the passed object is an actual date
+ * with an accepted format
+ *
+ * Allowed formats : http://dygraphs.com/date-formats.html
+ *
+ * TEAMMATES currently follows the RFC2822 / IETF date syntax
+ * e.g. 02 Apr 2012, 23:59
+ *
+ * @param date
+ * @returns boolean
+ */
+function isDate(date) {
+    return !isNaN(Date.parse(date));
+}
+
+/**
+* Function to test if param is a numerical value
+* @param num
+* @returns boolean
+*/
+function isNumber(num) {
+    return (typeof num === 'string' || typeof num === 'number') && !isNaN(num - 0) && num !== '';
```

Comment:
For future work: Is there a better way to do this?
Also, if not, perhaps this should go into multiple lines.

But don't bother now.